internal/pkg/plan/persist_test.go

```diff
@@ -248,11 +247,20 @@ func TestPersistNewConfigRow(t *testing.T) {
 		},
 		Path:        "extractor/keboola.ex-db-mysql/new-config",
 		ProjectPath: "main/extractor/keboola.ex-db-mysql/new-config",
-		Rows:        []*NewRowAction{rowAction},
 	}
+
+	// Delete callbacks for easier comparison (we only check callbacks result)
+	for _, action := range plan.actions {
```

Comment:
V testoch pred porovnanim obsahu `plan` odstranim `callbacks`, lebo tie by sa blbo porovnavali.

V teste sa najprv testuje obsah planu, no potom aj jeho vykonanie, takze to je dostatocne, ... je zarucene, ze danym `rows` sa ID nastavi.